win32_threads.c

```diff
@@ -377,10 +377,10 @@ GC_INLINE LONG GC_get_max_thread_index(void)
   static GC_bool isWow64; /* Is running 32-bit code on Win64?   */
 # define GET_THREAD_CONTEXT_FLAGS (isWow64 \
                         ? CONTEXT_INTEGER | CONTEXT_CONTROL \
-                          | CONTEXT_EXCEPTION_REQUEST | CONTEXT_SEGMENTS \
-                        : CONTEXT_INTEGER | CONTEXT_CONTROL)
+                          | CONTEXT_EXCEPTION_REQUEST | CONTEXT_SEGMENTS | CONTEXT_FLOATING_POINT\
```

Comment:
Please revert these 2 lines (the patch is not about 32-bit mode)

Reply:
It should look like this:
```
#ifdef WOW64_THREAD_CONTEXT_WORKAROUND
... // no change
#elif defined(I386)
# define GET_THREAD_CONTEXT_FLAGS (CONTEXT_INTEGER | CONTEXT_CONTROL)
#else
# define GET_THREAD_CONTEXT_FLAGS (CONTEXT_INTEGER | CONTEXT_CONTROL |  CONTEXT_FLOATING_POINT)
#endif /* X86_64 */
```

Reply:
> Please revert these 2 lines (the patch is not about 32-bit mode)

Done